src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -50,7 +52,7 @@ public PersonCard(Item item, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(item.getName().fullName);
         phone.setText(item.getId()); // id temporarily displayed in old phone label
-        //address.setText(person.getAddress().value);
+        address.setText(Integer.toString(item.getCount()));
```

Comment:
Since we're using the address label temporarily, why not just exclude `Label count` (line 44)?